### PR TITLE
Add-merchant-name-to-merchant-pages

### LINKED ISSUE #92: Add merchant name to merchant pages
To match the wireframe, add the merchant name in the header section to the left of the navbar.

### PR DESCRIPTION
**Pull request by MWoodshop**
# Describe the changes below:
- Adds merchant name in visitor partial header.
- Adds CSS for styling of merchant name.
- Removes save_and_open command from admin/merchants/index_spec

# Relevant user story:
Related to merchant wireframes not a specific user story.

# Specific feedback requests? Include file name and line:

# Before submitting, check the following:
- [X] Entire test suite is passing
- [X] SimpleCov test percentage (99.78%)

# Note any co-authors / anything else you'd like to add:

### CODE REVIEW

File: spec/features/admin/merchants/index_spec.rb [hunk 1]
```diff
@@ -76,7 +76,6 @@
 
     expect("Enabled Merchants").to appear_before("Disabled Merchants")
 
-    save_and_open_page
     within("#enabled-merchants") do
```
Comment (awiley33) [inline spec/features/admin/merchants/index_spec.rb:79]: thanks for removing that

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -240,3 +240,17 @@ ul {
...
+  align-items: center;
+}
+
+#merchant-name {
+  margin-left: 20px; /* adjust as needed */
+}
+
+#nav_bar {
+  margin-right: 20px; /* adjust as needed */
+}
```
Comment (awiley33) [inline app/assets/stylesheets/application.css:291]: thanks for designing this CSS

**Review by awiley33 — APPROVED** [review]
Looks good Mike!

**Review by MatthewTLim — APPROVED** [review]
Nice catch with the save and open page. Everything else looks great!

**Comment by westonio** [comment]
@MWoodshop This PR has been approved, but has some conflicts -- do you want to resolve these for us?

**Pull request merged**